gridData/OpenDX.py

```diff
@@ -459,6 +472,13 @@ def __init__(self,classid='0',components=None,comments=None):
         self.components = components
         self.comments= comments
 
+    def _openfile(self, filename):
```

Comment:
Rename to `_openfile_writing()` or similar to make sure that it is clear that this _only_ opens a file for writing. Otherwise one immediately wonders where the `"w"` kwarg is...

Reply:
done!